src/__tests__/error.spec.ts

```diff
@@ -24,7 +24,7 @@ globalThis.foo = {
 describe('The Error Boundary', () => {
     it('should preserve identity of errors after a membrane roundtrip', function() {
         expect.assertions(3);
-        const evalScript = createSecureEnvironment();
+        const evalScript = createSecureEnvironment(undefined, window);
```

Comment:
If the `window` is the required param then it should go first, with the optional distortion map param second.